scripts/helm-docs.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 ## Reference: https://github.com/norwoodj/helm-docs
 set -eux
-CHART_DIR="$(cd "$(dirname "$0")/.." && pwd)"
-echo "$CHART_DIR"
+REPO_ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+echo "$REPO_ROOT"
 
 echo "Running Helm-Docs"
 docker run \
-    -v "$CHART_DIR:/helm-docs" \
+    -v "$REPO_ROOT:/helm-docs" \
     -u $(id -u) \
     jnorwood/helm-docs:v1.9.1
```

Comment:
Not a blocker, but is this worth bumping as well? If available, though.

Reply:
Yes. Good idea.

Reply:
https://hub.docker.com/layers/jnorwood/helm-docs/v1.11.0/images/sha256-e438eb9f879e4bb8389ce4acd0f7e9193e3d62e877a1a3cd5abd0393dbe55fe5?context=explore

1.11.0

Reply:
If I remember right we haven't updated helm-docs as there is different behaviour for `##` comments that contain notes or related links. Newer versions outputs these values also into rendered README.md

Reply:
Happy to leave it as is if we're seeing issues.

Reply:
The file isn't included in any charts and therefore doesn't appear to need to be reflected in any versioning.

The change here also doesn't change the behavior for charts going forward.

Reply:
@pdrastil I was seeing comments coming through on the current version (v1.9.1) in the file.  Upgrading gave same effect.  I just pushed the new version but can roll back.

Reply:
Created #1784 to be grabbed later